loadgen/requirements.txt

```diff
@@ -0,0 +1,5 @@
+barnum==0.5.1
+kafka-python==2.0.2
+mysql-connector-python==8.0.27
+noise==1.2.2
+requests==2.27.1
```

Comment:
I pinned the versions of all the Python dependencies to prevent breakage if any of these packages issue breaking releases.